addons/stock/views/stock_quant_views.xml

```diff
@@ -494,6 +494,32 @@ in this location. That leads to a negative stock.
         </field>
     </record>
 
+    <!-- Update Quantity (via product) view -->
+    <record model="ir.ui.view" id="view_stock_quant_tree_inventory_editable_update_qty">
+        <field name="name">stock.quant.inventory.tree.editable.update.qty</field>
+        <field name="model">stock.quant</field>
+        <field eval="10" name="priority"/>
+        <field name="mode">primary</field>
+        <field name="inherit_id" ref="stock.view_stock_quant_tree_editable"/>
+        <field name="arch" type="xml">
+            <xpath expr="//button[@name='action_inventory_at_date']" position="attributes">
+                <attribute name="display">none</attribute>
```

Comment:
invisible=1 instead of display=none

Reply:
Greetings @Whenrow,

I have already implemented your suggested approach, but unfortunately, it did not yield the desired results. In the parent view, which is **view_stock_quant_tree_editable**, there is an attribute called display set to 'always'. Due to this, I utilized display='none' in the current scenario. Would you kindly advise whether I should maintain it as it is or if there is something else I should modify instead? 

Thank you.

Reply:
Do we really need a new view for it? You can not just use invisible if default_product_id key is in context

Reply:
Greetings @amoyaux 

Thank you for the response! We do require a separate view as some columns need to be hidden, and unfortunately, we cannot achieve this in the parent view as it is utilized in the location report as well.